var/spack/repos/builtin/packages/regiodesics/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Regiodesics(CMakePackage):
+    """Vector direction computation
+    """
+
+    homepage = "https://bbpteam.epfl.ch/project/spaces/display/BBPNSE/Computing+neurons+direction+vectors"
+    git = "ssh://bbpcode.epfl.ch/viz/Regiodesics"
+
+    version('master', branch='master', submodules=True)
```

Comment:
```suggestion
    version('develop', branch='master', submodules=True)
```

Reply:
https://github.com/spack/spack/pull/14915#discussion_r379210935

Reply:
I can also just nuke that version